app/poros/quote.rb

```diff
@@ -0,0 +1,9 @@
+class Quote
+  attr_reader :author, :content, :categories
+
+  def initialize(quote_data)
+    @author = quote_data[:author]
+    @content = quote_data[:content]
+    @categories = quote_data[:tags]
+  end  
```

Comment:
If you added the id here that came through from the API, you could use it to help you write good tests.